src/views/BookingPage.vue

```diff
@@ -0,0 +1,129 @@
+<script setup lang="ts">
+import { ref, computed } from "vue";
+import { useBookingStore } from "@/stores/booking";
+import { postReservation } from "@/api/reservationsService";
+import BookingTabs from "@/components/bookings/BookingTabs.vue";
+import BookingHeader from "@/components/bookings/BookingHeader.vue";
+import SeatingGrid from "@/components/bookings/SeatingGrid.vue";
+import BaseButton from "@/components/base/BaseButton.vue";
+import BaseSelect from "@/components/base/BaseSelect.vue";
+import BaseInput from "@/components/base/BaseInput.vue";
+import { ticketIds } from "@/api/ticketID";
+const bookingStore = useBookingStore();
+
+const selectedTab = ref("Choose seats");
+
+const headerTitle = computed(() => {
+  return `Choose your ${
+    selectedTab.value === "Choose seats" ? "seats" : "tickets"
+  }`;
+});
```

Comment:
I think this could be done in template, what do you think?

Reply:
I like putting things like this in template, I did encounter few comments suggesting use of computed though